routes/index.tsx

```diff
@@ -9,24 +9,47 @@ import {
   getAllItems,
   getItemCommentsCount,
   getUsersByIds,
+  getVotesByUser,
   type Item,
   type User,
+  type Vote,
 } from "@/utils/db.ts";
 
 interface HomePageData extends State {
   users: User[];
   items: Item[];
   commentsCounts: number[];
+  votes: Vote[];
 }
 
+/** @todo Accept property other than score */
+export const sortItems = (items: Item[], property: "score") => {
+  const fn = (a: Item, b: Item) => {
+    const x = Number(a[property]);
+    const y = Number(b[property]);
+    if (x > y) {
+      return -1;
+    }
+    if (x < y) {
+      return 1;
+    }
+    return 0;
+  };
+  return items.sort(fn);
+};
+
```

Comment:
Sorry, I should've been clearer. Can you please move the `compareFn` out into its own function? For now, I suggest  `compareScore()` and removing the `property` parameter. We can expand functionality later if needed.